### PR TITLE
fix: container widget height with title

### PR DESCRIPTION
**Pull request by jake-bassett**

### CODE REVIEW

File: projects/dashboards/src/widgets/container/container-widget-renderer.component.scss [hunk 1]
```diff
@@ -8,7 +8,16 @@
   flex-direction: column;
 
   .title {
+    height: 28px;
     @include widget-title($gray-9);
     padding-bottom: 16px;
   }
+
+  .content {
```
Comment (anandtiwary) [inline projects/dashboards/src/widgets/container/container-widget-renderer.component.scss:16]: `flex: 1 1 auto` for .content would also do the same right?

Comment (anandtiwary) [reply projects/dashboards/src/widgets/container/container-widget-renderer.component.scss:16]:
```
.title {
height: 28px
}

.content {
flex: 1 1 auto;
}
```

Comment (jake-bassett) [reply projects/dashboards/src/widgets/container/container-widget-renderer.component.scss:16]: That was my first attempt as well. I can't get `flex` to work here for some reason. When I try that, the `.content` just expands to fit its... well, content.

I could probably circle back to this, but this was the quickest/easiest solution without worrying about far reaching side effects.

Comment (anandtiwary) [reply projects/dashboards/src/widgets/container/container-widget-renderer.component.scss:16]: okay.. we can merge it if you are blocked. I can also play around with it later today.

**Review by anandtiwary — APPROVED** [review]

**Pull request merged**